src/java.base/share/classes/java/security/Signature.java

```diff
@@ -1507,7 +1508,9 @@ protected void engineUpdate(byte[] b, int off, int len)
         protected byte[] engineSign() throws SignatureException {
             try {
                 return cipher.doFinal();
-            } catch (IllegalBlockSizeException | BadPaddingException e) {
```

Comment:
Is this a revert of a recent fix from someone else?

Reply:
Must be. Thanks